tests/bits/parameter_handler_backslash_03.expect=diff.output

```diff
@@ -0,0 +1,2 @@
+
+DEAL::a, b, c
```

Comment:
why is the file named this way? why not just `parameter_handler_backslash_03.output`?